scripts/globals/mobskills/sub-zero_smash.lua

```diff
@@ -0,0 +1,35 @@
+-----------------------------------
+--  Sub-Zero Smash
+--
+--  Description: Additional Effect: Paralysis. Damage varies with TP.
+--  Type: Physical (blunt)
+--  Range: Cone (5' yalms)
+--  Notes: This spell should be used anytime the target is behind the mob.
+--         However the online documentation suggests that this spell can
+--         still be used anytime. As a result, any other Ruszor spells
+--         should not trigger if the target is behind the mob.
+-----------------------------------
+require("scripts/globals/settings")
```

Comment:
Doesn't look like this settings require is used.

I tend to do widest line 1st unless I have a logic reason not to so:
```lua
-----------------------------------
require("scripts/globals/monstertpmoves")
require("scripts/globals/status")
-----------------------------------
```
?

Reply:
I was going by the (now many-times-deprecated) docs from Darkstar: https://wiki.dspt.info/index.php/How_to_Add_TP_Moves

```
Always:

require("/scripts/globals/settings");
require("/scripts/globals/status");
require("/scripts/globals/monstertpmoves");
```

I'm happy to remove unused requires.

I typically favor alphabetizing the imports, but I'm open to list them in an order here. I don't have a strong preference.

Reply:
I've been murdering extra requires every chance I get for a long time now, but old docs have copypasta from old files unfortunately and the current project wiki needs lots of love still.

I don't think we have any rule on ordering other then where multiple diff things (mixins, regular requires, other types) or concerned. My OCD only really cares that we get _one_  pattern and stick with it. I dunno how many are sorted which way currently.

Reply:
I tend to prefer alpha as well, though my personal approach is usually tree location, then alpha after.  I'm fine with any consistent approach so long as we only require things that are, well, required.

Reply:
tree then alpha sounds logical if we have anyone willing to do some mass editing later. for this pr for now: anythings acceptable we'll clean up later. I just took a quick glance and its gonna be a few hundred scripts needing cleanup for copyapasta

Reply:
I'll double check that it's safe to remove 'settings.lua'.

Otherwise I'll leave the order the same for now (since it's consistent). We can write a script to sort the requires across the whole codebase.

Reply:
Just one note, would suggest not bulk sorting everything; there's a few instances where items are broken out on purpose (for example, quests and missions that use zone IDs)

Reply:
mobskill subdir should be safe to do though